.github/workflows/workflow.yml

```diff
@@ -19,12 +19,48 @@ jobs:
         with:
           path: poetry.lock
           key: ${{ github.sha }}-${{ matrix.python-version }}
-  pytest:
+  pytest-dynalite:
     needs: lock
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
     runs-on: "ubuntu-latest"
+    services:
+      dynalite:
+        image: dimaqq/dynalite:latest
```

Comment:
Hey folks, I've moved the image to ghcr.io though it's documented poorly 😅 
https://github.com/dimaqq/dynalite-docker